Formula/libzt.rb

```diff
@@ -0,0 +1,36 @@
+class Libzt < Formula
+  desc "Encrypted P2P networking library for applications"
+  homepage "https://www.zerotier.com"
+
+  url "https://github.com/zerotier/libzt.git",
+    :branch   => "master",
```

Comment:
This needs to be a tag.

Reply:
Changed to a tag generated by `git describe` since I didn't want to increment my official version because of compliance changes. I hope this is acceptable.